aten/src/ATen/native/SummaryOps.cpp

```diff
@@ -31,6 +31,7 @@ Tensor _bincount_cpu_template(
   }
 
   Tensor output;
+  int64_t self_size = self.size(0);
```

Comment:
It may lead to the implicit casting to `int64_t`. Would you please use `int64_t self_size = static_cast<int64_t>(self.size(0));` instead?